### PR TITLE
Update adopters.geojson

### PR DESCRIPTION
**Pull request by nadavkav**
Add Weizmann institute of science, science teaching department

### CODE REVIEW

File: adopters.geojson [hunk 1]
```diff
@@ -88,6 +87,7 @@
     {"type": "Feature", "properties": {"institution": "TU Delft, Faculty TBM"}, "geometry": {"type": "Point", "coordinates": [4.3694, 52.0025459 ]}},
     {"type": "Feature", "properties": {"institution": "RIADVICE"}, "geometry": {"type": "Point", "coordinates": [10.2057799, 36.7088573]}},
     {"type": "Feature", "properties": {"institution": "PrivacyLx"}, "geometry": {"type": "Point", "coordinates": [-9.15808, 38.74410]}},
-    {"type": "Feature", "properties": {"institution": "Ventspils University of Applied Sciences"}, "geometry": {"type": "Point", "coordinates": [21.5461, 57.3848]}}
+    {"type": "Feature", "properties": {"institution": "Weizmann institute of science, science teaching department"}, "geometry": {"type": "Point", "coordinates": [34.811533, 31.907875]}},
+    {"type": "Feature", "properties": {"institution": "Ventspils University of Applied Sciences"}, "geometry": {"type": "Point", "coordinates": [21.5461, 57.3848]}},
```
Comment (lkiesow) [inline adopters.geojson:91]: The last entry may not have a comma:

```suggestion
    {"type": "Feature", "properties": {"institution": "Ventspils University of Applied Sciences"}, "geometry": {"type": "Point", "coordinates": [21.5461, 57.3848]}}
```

File: adopters.geojson [hunk 2]
```diff
@@ -31,7 +31,6 @@
     {"type": "Feature", "properties": {"institution": "Technical University of Applied Sciences Lübeck"}, "geometry": {"type": "Point", "coordinates": [10.7002646, 53.837048]}},
     {"type": "Feature", "properties": {"institution": "Hochschule Bochum, Campus Velbert/Heiligenhaus"}, "geometry": {"type": "Point", "coordinates": [6.96794, 51.32762]}},
     {"type": "Feature", "properties": {"institution": "Hochschule Bochum"}, "geometry": {"type": "Point", "coordinates": [7.27248, 51.44711]}},
-    {"type": "Feature", "properties": {"institution": "INSA Lyon"}, "geometry": {"type": "Point", "coordinates": [4.8759893530497695, 45.7830905]}},
```
Comment (lkiesow) [inline adopters.geojson:34]: You probably did not want to remove this institution, did you?

**Review by lkiesow — CHANGES_REQUESTED** [review]

File: adopters.geojson (hunk 2 above)
Comment (nadavkav) [reply adopters.geojson:34]: Right. and I do not remember I did. so It might be a mistake. please see if you can discard that part of the patch.

**Comment by tibroc** [comment]
To keep things simple, I just pushed your original line here: https://github.com/elan-ev/bigbluebutton-map/commit/9ca2891a15a32481ee24bf129ea763c189218f2d

I will close this PR with that.

**Pull request closed without merging**